azure-quantum/azure/quantum/target/rigetti/target.py

```diff
@@ -32,6 +32,30 @@ class RigettiTarget(str, Enum):
     ASPEN_11 = "rigetti.qpu.aspen-11"
     ASPEN_M_1 = "rigetti.qpu.aspen-m-1"
 
+    def simulators():
+        """Returns a list of simulator targets"""
+        return [
+            RigettiTarget.QVM.value,
+        ]
+
+    def qpus():
+        """Returns a list of QPU targets"""
+        return [
+            RigettiTarget.ASPEN_11.value,
+            RigettiTarget.ASPEN_M_1.value,
+        ]
```

Comment:
```suggestion
    def simulators() -> List[str]:
        """Returns a list of simulator targets"""
        return [
            RigettiTarget.QVM.value,
        ]

    def qpus() -> List[str]:
        """Returns a list of QPU targets"""
        return [
            RigettiTarget.ASPEN_11.value,
            RigettiTarget.ASPEN_M_1.value,
        ]
```

Does it matter that these are now lists when before I believe they were tuples?